src/util/generatePreview.js

```diff
@@ -119,21 +137,36 @@ class PreviewGenerator {
     this.previewBody.appendChild(scriptTag);
   }
 
+  _addAlertHandling() {
+    const scriptTag = this.previewDocument.createElement('script');
+    scriptTag.innerHTML = alertReplacementScript;
+    this.previewBody.appendChild(scriptTag);
+  }
+
   _attachLibraries() {
     this._project.enabledLibraries.forEach((libraryKey) => {
       const library = libraries[libraryKey];
-      const css = library.css;
-      const javascript = library.javascript;
-      if (css !== undefined) {
-        castArray(css).forEach(this._attachCssLibrary.bind(this));
-      }
-      if (javascript !== undefined) {
-        castArray(javascript).
-          forEach(this._attachJavascriptLibrary.bind(this));
-      }
+      this._attachLibrary(library);
+    });
+
+    Object.keys(alwaysOnLibraries).forEach((libraryKey) => {
+      const library = alwaysOnLibraries[libraryKey];
+      this._attachLibrary(library);
```

Comment:
Ack, sorry, I didn’t think of this before, **but**: I think we only want to load `sweetalert` if we’re also adding alert handling (i.e. neither should exist in the pop-out environment). So, rather than having the concept of “always on”, I think it’s more a situation of “internal libraries that are available to `generatePreview` to use as needed”. This makes the `_attachLibrary` method even more useful, and also cuts in favor of the approach here you took of keeping sweetalert separate from the user-toggled libraries. Thoughts?